core/src/main/java/io/questdb/griffin/WhereClauseParser.java

```diff
@@ -455,10 +463,18 @@ private boolean analyzeInInterval(IntrinsicModel model, ExpressionNode col, Expr
             // Single value ts in '2010-01-01' - treat string literal as an interval, not single Timestamp point
             ExpressionNode inArg = in.rhs;
             if (inArg.type == ExpressionNode.CONSTANT) {
-                if (!isNegated) {
-                    model.intersectIntervals(inArg.token, 1, inArg.token.length() - 1, inArg.position);
+                if (isNullKeyword(inArg.token)) {
+                    if (!isNegated) {
+                        model.intersectIntervals(Numbers.LONG_NaN, Numbers.LONG_NaN);
+                    } else {
+                        model.subtractIntervals(Numbers.LONG_NaN, Numbers.LONG_NaN);
```

Comment:
do nothing, fall throug to lines 

``
in.intrinsicValue = IntrinsicModel.TRUE;
return true;
``